README.md

```diff
@@ -144,8 +139,11 @@ public static async Task SampleMethodAsync()
         dbTracer.End();
     }
 }
+
 ```
 
+>*Note*: Previous versions of the OneAgent SDK supported tracing of asynchronous methods (which are C# methods that are marked with the `async` keyword) using the method `StartAsync()`. This method has been deprecated for technical reasons and the recommended way of tracing asynchronous code are the `TraceAsync` methods (see below).
```

Comment:
"for technical reasons" not needed. Either you explain the reasons or you don't ;)
It is enough to say it has been deprecated.